client/post-editor/media-modal/gallery/preview.jsx

```diff
@@ -13,48 +16,47 @@ import EditorMediaModalGalleryEdit from './edit';
 import EditorMediaModalGalleryPreviewShortcode from './preview-shortcode';
 import EditorMediaModalGalleryPreviewIndividual from './preview-individual';
 
-export default React.createClass( {
-	displayName: 'EditorMediaModalGalleryPreview',
+/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
Ideally we can get rid of this in the future. Still a bit sad that these are errors vs warnings before we cleaned them up.

Reply:
Yah, I figured the disabler beats the rest for the time being, thanks for reviewing @gwwar!